NEWS

```diff
@@ -107,84 +127,114 @@ New Functionality
   by turning off event handlers at runtime that are only needed for log
   generation.
 
-- On Linux, the AF_PACKET packet source plugin (https://github.com/zeek/zeek-af_packet-plugin)
-  is included as builtin plugin by default. To select this packet source, prefix
-  the interface name with ``af_packet``.
+- On Linux, the AF_PACKET packet source plugin
+  (https://github.com/zeek/zeek-af_packet-plugin) is included as a builtin
+  plugin by default. To select this packet source, prefix the interface name
+  with ``af_packet``.
 
 	zeek -i af_packet::eth0
 
 - Usage of ``break`` and ``next`` statements is now validated. It was previously
-  possible to place these outside of ``for``, ``while`` or ``switch``
-  statements without any error indication.
+  possible to place these outside of ``for``, ``while`` or ``switch`` statements
+  without any error indication.
 
-- Add two BIFs ``get_identifier_declaring_script()`` and ``get_record_field_declaring_script()``
-  to query the declaring scripts for identifiers and record fields from Zeek scripts.
+- Add two BIFs ``get_identifier_declaring_script()`` and
+  ``get_record_field_declaring_script()`` to query the declaring scripts for
+  identifiers and record fields from Zeek scripts.
 
 - Extend the SSH analyzer to produce new events (``ssh2_ecc_init``,
-  ``ssh2_gh_gex_init``, ``ssh2_gss_init``, ssh2_rsa_secret``) to detect
-  when SSH client and server roles are reversed.
+  ``ssh2_gh_gex_init``, ``ssh2_gss_init``, ssh2_rsa_secret``) to detect when SSH
+  client and server roles are reversed.
 
 - Analyzers found in the new ``Analyzer::requested_analyzers`` set will be
-  enabled at ``zeek_init()`` time. The set can be populated via :zeek:see:`redef`.
-  This change only has an effect in settings where ``Analyzer::disable_all``
-  is changed to ``T``. By default, all analyzers continue to be enabled.
-
-- A new ``analyzer.log`` was added to log all analyzer violations and
-  optionally analyzer confirmations. This log can be useful during development
-  of new analyzers as well as for collecting operational data in production
+  enabled at ``zeek_init()`` time. The set can be populated via
+  :zeek:see:`redef`.  This change only has an effect in settings where
+  ``Analyzer::disable_all`` is changed to ``T``. By default, all analyzers
+  continue to be enabled.
+
+- A new ``analyzer.log`` was added to log all analyzer violations and optionally
+  analyzer confirmations. This log can be useful during development of new
+  analyzers as well as for collecting operational data in production
   environments.
 
+- A new ``join_string_set`` BIF was added, replacing the existing script-level
+  version from utils/strings.zeek.
+
+- A new ``&ordered`` attribute for tables and sets was added. This attribute
+  causes iteration over a table/set to return elements in the order of their
+  insertion.
+
+- A new ``-D`` argument was added to the ``configure`` script to allow passing
```

Comment:
Is this Zeek NEWS worthy?

Reply:
I could go either way. It's notable since we removed the option for CAF, and that we're already calling out a number of other additions to that script, but I could also see it being considered a more-technical detail that most users won't care about.

Reply:
Up to you then :-)